dashboard/lib/policies/lti.rb

```diff
@@ -146,7 +146,7 @@ def self.lti?(user)
   end
 
   def self.only_lti_auth?(user)
-    user.authentication_options&.length == 1 && user.authentication_options.first.lti?
+    user.authentication_options.any? && user.authentication_options.all?(&:lti?)
```

Comment:
```suggestion
    user.authentication_options.all?(&:lti?)
```

Reply:
@artem-vavilov I think we want them to have at least 1 auth option, otherwise we wouldn't consider them LTI. `.all` will return true for an empty array:

```
[development] dashboard > u.authentication_options
=> []
[development] dashboard > u.authentication_options.all?(&:lti?)
=> true
```

Reply:
Yes, this is what I found initially after Wilkie suggested using `.all`.